src/l500/l500-options.h

```diff
@@ -30,31 +30,37 @@ namespace librealsense
     };
 
     class l500_options;
+    class l500_hw_options;
 
     typedef uvc_xu_option< int > super;
 
     class digital_gain_option : public super
     {
     public:
-        digital_gain_option( uvc_sensor & ep,
-                             platform::extension_unit xu,
-                             uint8_t id,
-                             std::string description,
-                             const std::map< float, std::string > & description_per_value,
-                             firmware_version fw_version,
-                             l500_options * owner )
+        digital_gain_option(
+            uvc_sensor & ep,
+            platform::extension_unit xu,
+            uint8_t id,
+            std::string description,
+            const std::map< float, std::string > & description_per_value,
+            firmware_version fw_version,
+            l500_options * owner,
+            std::map< rs2_option, std::shared_ptr< cascade_option< l500_hw_options > > >
```

Comment:
Can we typedef this ugly shared_ptr somehow to make it more readable?

Reply:
done